packages/@aws-cdk/assets-docker/lib/image-asset.ts

```diff
@@ -10,6 +10,15 @@ export interface DockerImageAssetProps {
    * The directory where the Dockerfile is stored
    */
   directory: string;
+
+  /**
+   * ECR repository name, if omitted a dynamic name is used instead.
+   * Specify this property if you need to statically address the
+   * image, e.g. from a Kubernetes Pod.
+   * Note, this is only the repository name, without the registry and
+   * the tag parts.
+   */
+  repositoryName?: string;
```

Comment:
Please add `@default` tag.

Reply:
What is the default value of an optional property, `null`, `unedfined` or just no value?

Reply:
In `@default` we don't write the literal value that would be assigned to it by the typesystem; instead, we write the behavior that the user would see if they didn't supply the value.